pkg/csi/service/common/commonco/k8sorchestrator/k8sorchestrator.go

```diff
@@ -225,18 +234,122 @@ func initFSS(ctx context.Context, k8sClient clientset.Interface, controllerClust
 			fssConfigMap, err = k8sClient.CoreV1().ConfigMaps(k8sOrchestratorInstance.internalFSS.configMapNamespace).Get(
 				ctx, k8sOrchestratorInstance.internalFSS.configMapName, metav1.GetOptions{})
 			if err != nil {
-				log.Errorf("failed to fetch configmap %s from namespace %s. Setting the internal feature states to default values: %v. Error: %v",
-					k8sOrchestratorInstance.internalFSS.configMapName, k8sOrchestratorInstance.internalFSS.configMapNamespace,
-					k8sOrchestratorInstance.internalFSS.featureStates, err)
-			} else {
-				// Update values
-				k8sOrchestratorInstance.internalFSS.featureStates = fssConfigMap.Data
-				log.Infof("New internal feature states values stored successfully: %v", k8sOrchestratorInstance.internalFSS.featureStates)
+				// return error as we cannot init containers without this info
+				log.Errorf("failed to fetch configmap %s from namespace %s. Error: %v",
+					k8sOrchestratorInstance.internalFSS.configMapName, k8sOrchestratorInstance.internalFSS.configMapNamespace, err)
+				return err
 			}
+			// Update values
+			k8sOrchestratorInstance.internalFSS.featureStates = fssConfigMap.Data
+			log.Infof("New internal feature states values stored successfully: %v", k8sOrchestratorInstance.internalFSS.featureStates)
 		}
 	}
 
-	// Set up kubernetes resource listeners for k8s orchestrator
+	if controllerClusterFlavor == cnstypes.CnsClusterFlavorGuest && serviceMode != "node" {
```

Comment:
Do we have k8s orchestrator for node daemonset too?

Reply:
Yes online resize started using `IsFSSEnabled` API, therefore we need to handle this.